src/openforms/products/models/product.py

```diff
@@ -20,7 +22,12 @@ class Product(models.Model):
         unique=True,
     )
     name = models.CharField(_("name"), max_length=50)
-    price = models.DecimalField(_("price"), max_digits=10, decimal_places=2)
+    price = models.DecimalField(
+        _("price"),
+        max_digits=10,
+        decimal_places=2,
+        validators=[MinValueValidator(Decimal("0.01"))],
+    )
```

Comment:
I think a validator that does `value > Decimal("0.0")` would be better. It's a shame that Django does not include such a validator (it only does _greater than or equal_), you can define it like so:

```python
@deconstructible
class StrictMinValueValidator(MinValueValidator):
    message = _('Ensure this value is greater than %(limit_value)s.')
    code = 'strict_min_value'

    def compare(self, a, b):
        return a <= b
```
However this requires translations for the message etc. and I don't think we'll see a case where people would want prices between 0 and 0.01 so it's also fine as is.